code/game/mecha/working/ripley.dm

```diff
@@ -55,10 +57,13 @@
 		var/obj/item/mecha_parts/mecha_equipment/tool/drill/D = new /obj/item/mecha_parts/mecha_equipment/tool/drill
 		D.attach(src)
 
-	//Attach hydrolic clamp
+	//Attach hydraulic clamp
 	var/obj/item/mecha_parts/mecha_equipment/tool/hydraulic_clamp/HC = new /obj/item/mecha_parts/mecha_equipment/tool/hydraulic_clamp
 	HC.attach(src)
-	for(var/obj/item/mecha_parts/mecha_tracking/B in src.contents)//Deletes the beacon so it can't be found easily
+	src.hydraulic_clamp = HC
+
+	//Deletes the beacon so it can't be found easily
+	for(var/obj/item/mecha_parts/mecha_tracking/B in src.contents)

```

Comment:
If you're bored though add an argument to new() in working.dm for ripleys so this tracking beacon isn't spawned in the first place, what the fudge.